### ISSUE TITLE
training loss became nan

### ISSUE THREAD

**Issue by d5423197** [AlexeyAB/darknet#8459]
Hello Author,

I have about 1500 original images (2040  x 2040). I applied rotation on it. So the training set is about 30k.
After 1253 epochs, the training loss is about 5. But training loss became -nan at epoch 1254.

I just what to know why this happen and how to avoid it. 
Thanks for your help
